### ISSUE TITLE
My model doesn't train well when updating to 0.8

### ISSUE THREAD

**Issue by sunshineatnoon** [Theano/Theano#4119]
I trained a model on Theano 0.7 and it works fine. The loss function looks like this:

![Theano 0.7](https://raw.githubusercontent.com/sunshineatnoon/sunshineatnoon.github.io/master/images/loss_theano7.jpg)
But when I updated to Theano 0.8, my model didn't train well like before, it seems the loss can't keep decreasing after just 2,3 epochs. The loss function looks like this:

![Theano 0.8](https://raw.githubusercontent.com/sunshineatnoon/sunshineatnoon.github.io/master/images/loss_theano8.jpg)
 And when I downgraded to Theano 0.7, my model trains well again.

**Comment by nouiz** [comment]
Can you do a git bisect to identify more when this problem got introduced
in Theano?

**Comment by sunshineatnoon** [comment]
@nouiz Thanks for helping, I found the problem, it's my misuse of Keras, I am so sorry about this and I will close thie issue, thanks!

**Comment by nouiz** [comment]
thanks for the update.
